src/openforms/logging/logevent.py

```diff
@@ -106,11 +118,13 @@ def pdf_generation_skip(submission: "Submission", submission_report):
 # - - -
 
 
-def prefill_retrieve_success(submission: "Submission", plugin):
+def prefill_retrieve_success(submission: "Submission", plugin, prefill_fields):
     _create_log(
         submission,
         "prefill_retrieve_success",
+        extra_data={"prefill_fields": prefill_fields},
         plugin=plugin,
+        tag_avg=True,
```

Comment:
Can we make this more generic: Instead of having a parameter `tag_avg=True` we could do something like `tags=[TimelineLogTags.AVG]`?

I'm not sure if this will work in querysets though, like [here](https://github.com/open-formulieren/open-forms/pull/715/files#diff-2b3c27113b65828e17d17771483ffba20f8327d8449195d7fe02ed743cd0332cR133)